pacbio-subreads-processing.wdl

```diff
@@ -21,175 +21,224 @@ version 1.0
             "biowdl-input-converter": "quay.io/biocontainers/biowdl-input-converter:0.2.1--py_0",
-            "ccs": "quay.io/biocontainers/pbccs:4.2.0--1",
+            "ccs": "quay.io/biocontainers/pbccs:5.0.0--0",
             "fastqc": "quay.io/biocontainers/fastqc:0.11.9--0",
             "isoseq3": "quay.io/biocontainers/isoseq3:3.3.0--0",
             "lima": "quay.io/biocontainers/lima:1.11.0--0",
-            "multiqc": "quay.io/biocontainers/multiqc:1.9--pyh9f0ad1d_0"
+            "python3": "python:3.7-slim",
+            "multiqc": "quay.io/biocontainers/multiqc:1.9--pyh9f0ad1d_0",
+            "pacbio-merge": "lumc/pacbio-merge:0.2"
         }
     }
 
     meta {allowNestedInputs: true}
 
-    SubreadsConfig subreadsConfig = read_json(subreadsConfigFile)
-    Array[Subreads] allSubreads = subreadsConfig.subreads
+    # The name of the subreads, to be used to determine output filenames
+    File subreadsName = basename(subreadsFile, ".subreads.bam")
+
+    # Get the CCS chunks
+    call ccsChunks as createChunks {
+        input:
+            chunkCount = ccsChunks,
+            dockerImage = dockerImages["python3"]
+    }
+
+    # Index the input bamfile
+    if (!defined(subreadsIndexFile)) {
+        call pbbam.Index as pbindex {
+            input:
+                bamFile = subreadsFile
+        }
+    }
+
+    File subreadsIndex = select_first([pbindex.index, subreadsIndexFile])
+    File subreadsBam = select_first([pbindex.indexedBam, subreadsFile])
+
+    scatter (chunk in createChunks.chunks) {
+        # Convert chunk from 1/10 to 1 to determine output filename
+        String chunkNumber = sub(chunk, "/.*$", "")
 
-    scatter (subreads in allSubreads) {
         call ccs.CCS as ccs {
             input:
-                subreadsFile = subreads.subreads_file,
-                outputPrefix = outputDirectory + "/" + subreads.subreads_id + "/" + subreads.subreads_id,
+                subreadsFile = subreadsBam,
+                subreadsIndexFile = subreadsIndex,
+                outputPrefix = subreadsName + "_chunk" + chunkNumber,
                 cores = ccsCores,
+                chunkString = chunk,
                 dockerImage = dockerImages["ccs"]
         }
+    }
+    # Merge the bam files again
+    call samtools.Merge as merge {
+        input:
+            bamFiles = ccs.ccsBam,
+            outputBamPath = subreadsName + ".ccs.bam"
+    }
 
-        call lima.Lima as lima {
-            input:
-                libraryDesign = libraryDesign,
-                ccsMode = ccsMode,
-                splitBamNamed = splitBamNamed,
-                inputBamFile = ccs.ccsBam,
-                barcodeFile = subreads.barcodes_file,
-                outputPrefix = outputDirectory + "/" + subreads.subreads_id + "/" + subreads.subreads_id,
-                cores = limaCores,
-                dockerImage = dockerImages["lima"]
-        }
+    # Merge the report for MultiQC
+    call mergePacBio as MergeCCSReport {
```

Comment:
```suggestion
    call mergePacBio as mergeCCSReport {
```

https://biowdl.github.io/styleGuidelines.html#5-naming-conventions
Biowdl calls should be lowerCamelCase ;)